### PR TITLE
[BULK UPDATES] DocuTune - Fix build validation issues: docs-link-absolute

### PR DESCRIPTION
**Pull request by alexbuckgit**
[BULK UPDATE] DocuTune - Fix build validation issues: docs-link-absolute

**Global effort to fix build validation errors**

The Microsoft Skilling team is fixing build validation errors and specific brand inconsistencies in Microsoft technical documentation for the rest of FY23 Q1. This will help address various accessibility, security, and usability issues. This PR was generated using DocuTune. It includes only targeted fixes and minor formatting adjustments.

DocuTune PRs improve quality of technical content by finding and automatically correcting a broad set of issues only where the correction is suitable. Please review within five business days and merge or comment in the PR with any changes you'd like to see. Thank you!

CorrelationId: 0df9c2d6-63f8-4d1e-8b57-b33611a06514
InitiativeId: docutune-build-validation-issues-2022-05
PointOfContact: Alex Buck

### CODE REVIEW

**Comment by alexbuckgit** [comment]
#docutune-draft

**Comment by JustPies** [comment]
#docutune-ready

**Comment by alexbuckgit** [comment]
@lindalu-MFST

Please merge the changes in this PR. For more information, see the description at the top of this PR.

Any pre-existing build validation issues or insufficient Acrolinx scores should be addressed in a separate PR.

#docutune-request-approval

**Comment by alexbuckgit** [comment]
@lindalu-MFST

Please merge the changes in this PR. For more information, see the description at the top of this PR.

Any pre-existing build validation issues or insufficient Acrolinx scores should be addressed in a separate PR.

#docutune-request-approval

File: Skype/Skype-For-Business-Bot-Framework/README.md [hunk 1]
```diff
@@ -1,26 +1,24 @@
...
+This documentation is published at [Skype Developer platform documentation](./docs/overview.md). Please read it there.
 
 If you're interested in contributing to this documentation, then please read on.
 
 ## Contribute to the documentation
 
-You may want to contribute new content or improve existing **Skype for Business Bot Framework** content. If you do, then clone this repository, 
-create a new branch from the main branch, add your content, and then create a pull request to contribute your changes back into the main branch.
-Pull requests are the way to move changes from a topic branch back into the main branch.
+You may want to contribute new content or improve existing **Skype for Business Bot Framework** content. If you do, then clone this repository, create a new branch from the main branch, add your content, and then create a pull request to contribute your changes back into the main branch. Pull requests are the way to move changes from a topic branch back into the main branch.
```
Comment (lindalu-MSFT) [inline Skype/Skype-For-Business-Bot-Framework/README.md:16]:
```suggestion
You may want to contribute new content or improve existing **Skype for Business Bot Framework** content. If you do, then clone this repository, create a new branch from the main branch, add your content, and then create a pull request to contribute your changes back into the main branch. Pull requests are the way to move changes from a topic branch back into the main branch.

>[!NOTE]
> We review this repo for pull requestions throughout the week, but if you need immediate assistance, or would like an editorial review, add lindalu-msft to the PR either by tagging in the comments or adding as a reviewer.
```

File: Skype/Trusted-Application-API/samples/AnonMeetingJoinSamples/README.md [hunk 2]
```diff
@@ -4,45 +4,59 @@ description: The Trusted Application API is a Rest API that enables developers t
...
 - Creation of an adhoc meeting URL
 
 The sample Web SDK application interacts with the Trusted Application sample and contains two demos:
+
 - Getting an anon application token from the Trusted Application
 - Getting an adhoc meeting URL from the Trusted Application
 - Using the anon application token to join Anonymously into a Skype for Business Online meeting
 - Enabling audio/video in an anonymously-joined Skype for Business Online meeting
 
 ## Getting started
```
Comment (lindalu-MSFT) [inline Skype/Trusted-Application-API/samples/AnonMeetingJoinSamples/README.md:22]:
```suggestion
```

Comment (lindalu-MSFT) [reply Skype/Trusted-Application-API/samples/AnonMeetingJoinSamples/README.md:22]: This heading adds no value.

File: Skype/Trusted-Application-API/samples/AnonMeetingJoinSamples/README.md [hunk 3]
```diff
@@ -4,45 +4,58 @@ description: The Trusted Application API is a Rest API that enables developers t
...
-6.  Deploy the the client (anonymous webpage) code contained in the `WebsiteSamples` folder to a web server (e.g. IIS on localhost, or Azure App Service to deploy to *.azurewebsites.net)
-7.  Create Azure storage used to save event messages from Platform Service, where you will get an account name and account key. Refer to [this link](https://azure.microsoft.com/documentation/articles/storage-create-storage-account/) for details.
-8.  Create Azure service bus for the process of saving the event messages to storage, from where you will get connection string. Refer to [this link](https://azure.microsoft.com/documentation/articles/service-bus-dotnet-get-started-with-queues/) for details.
 
+1. Download [Azure SDK v2.9](https://go.microsoft.com/fwlink/?LinkId=746481) or above.
```
Comment (lindalu-MSFT) [inline Skype/Trusted-Application-API/samples/AnonMeetingJoinSamples/README.md:25]:
```suggestion
1. Download [Azure SDK v2.9](https://go.microsoft.com/fwlink/?LinkId=746481).
```

Comment (lindalu-MSFT) [reply Skype/Trusted-Application-API/samples/AnonMeetingJoinSamples/README.md:25]: We don't use "above" to refer to other versions. Please specify if you're referring to all previous versions, or later versions.

File: Skype/Trusted-Application-API/samples/AnonMeetingJoinSamples/README.md [hunk 4]
```diff
@@ -4,45 +4,58 @@ description: The Trusted Application API is a Rest API that enables developers t
...
-6.  Deploy the the client (anonymous webpage) code contained in the `WebsiteSamples` folder to a web server (e.g. IIS on localhost, or Azure App Service to deploy to *.azurewebsites.net)
-7.  Create Azure storage used to save event messages from Platform Service, where you will get an account name and account key. Refer to [this link](https://azure.microsoft.com/documentation/articles/storage-create-storage-account/) for details.
-8.  Create Azure service bus for the process of saving the event messages to storage, from where you will get connection string. Refer to [this link](https://azure.microsoft.com/documentation/articles/service-bus-dotnet-get-started-with-queues/) for details.
 
+1. Download [Azure SDK v2.9](https://go.microsoft.com/fwlink/?LinkId=746481).
+
+2. Create a Cloud Service from Azure and give it a name to reserve a `*.cloudapp.net` URL. Refer to [Azure Cloud Services documentation](/azure/cloud-services/) for details.
```
Comment (lindalu-MSFT) [inline Skype/Trusted-Application-API/samples/AnonMeetingJoinSamples/README.md:27]:
```suggestion
2. Create a Cloud Service from Azure and give it a name to reserve a `*.cloudapp.net` URL. For more details, see [Azure Cloud Services documentation](/azure/cloud-services/).
```

File: Skype/Trusted-Application-API/samples/AnonMeetingJoinSamples/README.md [hunk 5]
```diff
@@ -4,45 +4,58 @@ description: The Trusted Application API is a Rest API that enables developers t
...
-7.  Create Azure storage used to save event messages from Platform Service, where you will get an account name and account key. Refer to [this link](https://azure.microsoft.com/documentation/articles/storage-create-storage-account/) for details.
-8.  Create Azure service bus for the process of saving the event messages to storage, from where you will get connection string. Refer to [this link](https://azure.microsoft.com/documentation/articles/service-bus-dotnet-get-started-with-queues/) for details.
 
+1. Download [Azure SDK v2.9](https://go.microsoft.com/fwlink/?LinkId=746481).
+
+2. Create a Cloud Service from Azure and give it a name to reserve a `*.cloudapp.net` URL. For more details, see [Azure Cloud Services documentation](/azure/cloud-services/).
+
+   > The Trusted Application API requires the use of SSL and https for your cloud service.  You will need to create a DNS CName that points to your `*.cloudapp.net` cloud service to give it a custom domain. Refer to [Configuring a custom domain name for an Azure cloud service](/azure/cloud-services/cloud-services-custom-domain-name-portal) for details.
```
Comment (lindalu-MSFT) [inline Skype/Trusted-Application-API/samples/AnonMeetingJoinSamples/README.md:29]:
```suggestion
   The Trusted Application API requires the use of SSL and https for your cloud service.  You'll need to create a DNS CName that points to your `*.cloudapp.net` cloud service to give it a custom domain. For details, see [Configuring a custom domain name for an Azure cloud service](/azure/cloud-services/cloud-services-custom-domain-name-portal).
```

File: Skype/Trusted-Application-API/samples/AnonMeetingJoinSamples/README.md [hunk 6]
```diff
@@ -4,45 +4,58 @@ description: The Trusted Application API is a Rest API that enables developers t
 ---
 
 # About these samples
-The Trusted Application API is a Rest API that enables developers to build Skype for Business Online back-end communications services for the cloud.  The Trusted Application API Samples contain samples for the bankend Trusted Application and samples for how to interact with the trusted application from a client-side browser using the Skype Web SDK. 
+
+The Trusted Application API is a Rest API that enables developers to build Skype for Business Online back-end communications services for the cloud.  The Trusted Application API Samples contain samples for the backend Trusted Application and samples for how to interact with the trusted application from a client-side browser using the Skype Web SDK.
```
Comment (lindalu-MSFT) [inline Skype/Trusted-Application-API/samples/AnonMeetingJoinSamples/README.md:8]:
```suggestion
The Trusted Application API is a REST API that enables developers to build Skype for Business Online back-end communications services for the cloud.  The Trusted Application API Samples contain samples for the backend Trusted Application and samples for how to interact with the trusted application from a client-side browser using the Skype Web SDK.
```

File: Skype/Trusted-Application-API/samples/AnonMeetingJoinSamples/README.md [hunk 7]
```diff
@@ -4,45 +4,58 @@ description: The Trusted Application API is a Rest API that enables developers t
...
 
 # About these samples
-The Trusted Application API is a Rest API that enables developers to build Skype for Business Online back-end communications services for the cloud.  The Trusted Application API Samples contain samples for the bankend Trusted Application and samples for how to interact with the trusted application from a client-side browser using the Skype Web SDK. 
+
+The Trusted Application API is a REST API that enables developers to build Skype for Business Online back-end communications services for the cloud.  The Trusted Application API Samples contain samples for the backend Trusted Application and samples for how to interact with the trusted application from a client-side browser using the Skype Web SDK.
 
 The Trusted Application sample deploys to an Azure Cloud Service and:
+
 - Permits anonymous sign-in
 - Creation of an adhoc meeting URL
```
Comment (lindalu-MSFT) [inline Skype/Trusted-Application-API/samples/AnonMeetingJoinSamples/README.md:13]:
```suggestion
- Creates an adhoc meeting URL
```

File: Skype/Trusted-Application-API/samples/AnonMeetingJoinSamples/README.md [hunk 8]
```diff
@@ -4,45 +4,58 @@ description: The Trusted Application API is a Rest API that enables developers t
...
 
 The Trusted Application sample deploys to an Azure Cloud Service and:
+
 - Permits anonymous sign-in
-- Creation of an adhoc meeting URL
+- Creates an adhoc meeting URL
 
 The sample Web SDK application interacts with the Trusted Application sample and contains two demos:
+
 - Getting an anon application token from the Trusted Application
```
Comment (lindalu-MSFT) [inline Skype/Trusted-Application-API/samples/AnonMeetingJoinSamples/README.md:17]:
```suggestion
- Get an anon application token from the Trusted Application
```

File: Skype/Trusted-Application-API/samples/AnonMeetingJoinSamples/README.md [hunk 9]
```diff
@@ -4,45 +4,58 @@ description: The Trusted Application API is a Rest API that enables developers t
...
+
 - Permits anonymous sign-in
-- Creation of an adhoc meeting URL
+- Creates an adhoc meeting URL
 
 The sample Web SDK application interacts with the Trusted Application sample and contains two demos:
-- Getting an anon application token from the Trusted Application
+
+- Get an anon application token from the Trusted Application
 - Getting an adhoc meeting URL from the Trusted Application
```
Comment (lindalu-MSFT) [inline Skype/Trusted-Application-API/samples/AnonMeetingJoinSamples/README.md:18]:
```suggestion
- Get an adhoc meeting URL from the Trusted Application
```

**Review by lindalu-MSFT — COMMENTED** [review]
Made several edits and committed them so as to get this published! Please let me know if you have questions.

File: Skype/Trusted-Application-API/samples/AnonMeetingJoinSamples/README.md [hunk 10]
```diff
@@ -4,45 +4,58 @@ description: The Trusted Application API is a Rest API that enables developers t
...
-- Creation of an adhoc meeting URL
+- Creates an adhoc meeting URL
 
 The sample Web SDK application interacts with the Trusted Application sample and contains two demos:
-- Getting an anon application token from the Trusted Application
-- Getting an adhoc meeting URL from the Trusted Application
+
+- Get an anon application token from the Trusted Application
+- Get an adhoc meeting URL from the Trusted Application
 - Using the anon application token to join Anonymously into a Skype for Business Online meeting
```
Comment (lindalu-MSFT) [inline Skype/Trusted-Application-API/samples/AnonMeetingJoinSamples/README.md:19]:
```suggestion
- Use the anon application token to join Anonymously into a Skype for Business Online meeting
```

File: Skype/Trusted-Application-API/samples/AnonMeetingJoinSamples/README.md [hunk 11]
```diff
@@ -4,45 +4,58 @@ description: The Trusted Application API is a Rest API that enables developers t
...
 
+1. Download [Azure SDK v2.9](https://go.microsoft.com/fwlink/?LinkId=746481).
+
+2. Create a Cloud Service from Azure and give it a name to reserve a `*.cloudapp.net` URL. For more details, see [Azure Cloud Services documentation](/azure/cloud-services/).
+
+   The Trusted Application API requires the use of SSL and https for your cloud service.  You'll need to create a DNS CName that points to your `*.cloudapp.net` cloud service to give it a custom domain. For details, see [Configuring a custom domain name for an Azure cloud service](/azure/cloud-services/cloud-services-custom-domain-name-portal).
+
+3. Use the [quick registration tool](https://aka.ms/skypeappregistration) for registering Skype for Business Trusted Applications in Azure and Skype for Business Online, that eliminates the need to register an Application manually in Azure portal. Optionally, you can manually register your application in Azure Portal, where you will get a Client ID and set an App ID URI. Refer to [Registration in Azure Active Directory](https://github.com/OfficeDev/skype-docs/tree/master/Skype/Trusted-Application-API/docs/RegistrationInAzureActiveDirectory.md) for details.
```
Comment (lindalu-MSFT) [inline Skype/Trusted-Application-API/samples/AnonMeetingJoinSamples/README.md:31]:
```suggestion
3. Use the [quick registration tool](https://aka.ms/skypeappregistration) to register Skype for Business Trusted Applications in Azure and Skype for Business Online. This eliminates the need to register an application manually in Azure portal. Optionally, you can manually register your application in Azure Portal, where you'll get a Client ID and set an App ID URI. For details, see [Registration in Azure Active Directory](https://github.com/OfficeDev/skype-docs/tree/master/Skype/Trusted-Application-API/docs/RegistrationInAzureActiveDirectory.md).
```

File: Skype/Trusted-Application-API/samples/AnonMeetingJoinSamples/README.md [hunk 12]
```diff
@@ -4,45 +4,58 @@ description: The Trusted Application API is a Rest API that enables developers t
...
+
+4. Register Trusted Endpoints in a Skype for Business Online tenant using PowerShell. Refer to [Setting up a Trusted Application Endpoint](https://github.com/OfficeDev/skype-docs/tree/master/Skype/Trusted-Application-API/docs/TrustedApplicationEndpoint.md) for details.
+
+5. Provide consent: When the application is registered in AAD, it is registered in the context of a tenant.  For a tenant to use the Service Application, for example, when the application is developed as a multi-tenant application, it must be consented to by that tenant's admin. Refer to [Tenant Admin Consent](https://github.com/OfficeDev/skype-docs/tree/master/Skype/Trusted-Application-API/docs/TenantAdminConsent.md) for more details.
```
Comment (lindalu-MSFT) [inline Skype/Trusted-Application-API/samples/AnonMeetingJoinSamples/README.md:35]:
```suggestion
5. When the application is registered in AAD, it's registered in the context of a tenant. For a tenant to use the Service Application, for example, when the application is developed as a multi-tenant application, that tenant's admin must provide consent. For details, see [Tenant admin consent](https://github.com/OfficeDev/skype-docs/tree/master/Skype/Trusted-Application-API/docs/TenantAdminConsent.md).
```

File: Skype/Trusted-Application-API/samples/AnonMeetingJoinSamples/README.md [hunk 13]
```diff
@@ -4,45 +4,58 @@ description: The Trusted Application API is a Rest API that enables developers t
...
+
+4. Register Trusted Endpoints in a Skype for Business Online tenant using PowerShell. Refer to [Setting up a Trusted Application Endpoint](https://github.com/OfficeDev/skype-docs/tree/master/Skype/Trusted-Application-API/docs/TrustedApplicationEndpoint.md) for details.
+
+5. When the application is registered in AAD, it's registered in the context of a tenant. For a tenant to use the Service Application, for example, when the application is developed as a multi-tenant application, that tenant's admin must provide consent. For details, see [Tenant admin consent](https://github.com/OfficeDev/skype-docs/tree/master/Skype/Trusted-Application-API/docs/TenantAdminConsent.md).
+
+6. Deploy the the client (anonymous webpage) code contained in the `WebsiteSamples` folder to a web server (e.g. IIS on localhost, or Azure App Service to deploy to `*.azurewebsites.net`).
+
+7. Create Azure storage used to save event messages from Platform Service, where you will get an account name and account key. Refer to [this link](https://azure.microsoft.com/documentation/articles/storage-create-storage-account/) for details.
```
Comment (lindalu-MSFT) [inline Skype/Trusted-Application-API/samples/AnonMeetingJoinSamples/README.md:39]:
```suggestion
7. Create Azure storage used to save event messages from Platform Service, where you will get an account name and account key. For details, see [Create a storage account](/azure/storage/common/storage-account-create).
```

File: Skype/Trusted-Application-API/samples/AnonMeetingJoinSamples/README.md [hunk 14]
```diff
@@ -4,45 +4,58 @@ description: The Trusted Application API is a Rest API that enables developers t
...
+
+5. When the application is registered in AAD, it's registered in the context of a tenant. For a tenant to use the Service Application, for example, when the application is developed as a multi-tenant application, that tenant's admin must provide consent. For details, see [Tenant admin consent](https://github.com/OfficeDev/skype-docs/tree/master/Skype/Trusted-Application-API/docs/TenantAdminConsent.md).
+
+6. Deploy the the client (anonymous webpage) code contained in the `WebsiteSamples` folder to a web server (e.g. IIS on localhost, or Azure App Service to deploy to `*.azurewebsites.net`).
+
+7. Create Azure storage used to save event messages from Platform Service, where you will get an account name and account key. For details, see [Create a storage account](/azure/storage/common/storage-account-create).
+
+8. Create Azure service bus for the process of saving the event messages to storage, from where you will get connection string. Refer to [this link](https://azure.microsoft.com/documentation/articles/service-bus-dotnet-get-started-with-queues/) for details.
```
Comment (lindalu-MSFT) [inline Skype/Trusted-Application-API/samples/AnonMeetingJoinSamples/README.md:41]:
```suggestion
8. Create Azure Service Bus for the process of saving the event messages to storage, and getting the connection string. For details, see [Get started with Azure Service Bus queues (.NET)](/azure/service-bus-messaging/service-bus-dotnet-get-started-with-queues).
```

File: Skype/Trusted-Application-API/samples/AnonMeetingJoinSamples/README.md [hunk 15]
```diff
@@ -4,45 +4,58 @@ description: The Trusted Application API is a Rest API that enables developers t
...
+
+6. Deploy the the client (anonymous webpage) code contained in the `WebsiteSamples` folder to a web server (e.g. IIS on localhost, or Azure App Service to deploy to `*.azurewebsites.net`).
+
+7. Create Azure storage used to save event messages from Platform Service, where you will get an account name and account key. For details, see [Create a storage account](/azure/storage/common/storage-account-create).
+
+8. Create Azure Service Bus for the process of saving the event messages to storage, and getting the connection string. For details, see [Get started with Azure Service Bus queues (.NET)](/azure/service-bus-messaging/service-bus-dotnet-get-started-with-queues).
 
 ## Deployment
+
 Once you have satisfied the prerequisites, it is time to configure the Trusted Application source code and deploy it to the Azure Cloud Service created in step 2.
```
Comment (lindalu-MSFT) [inline Skype/Trusted-Application-API/samples/AnonMeetingJoinSamples/README.md:45]:
```suggestion
Once you've satisfied the prerequisites, it's time to configure the Trusted Application source code and deploy it to the Azure Cloud Service created in Step 2.
```

File: Skype/Trusted-Application-API/samples/AnonMeetingJoinSamples/README.md [hunk 16]
```diff
@@ -4,45 +4,58 @@ description: The Trusted Application API is a Rest API that enables developers t
...
-Once you have satisfied the prerequisites, it is time to configure the Trusted Application source code and deploy it to the Azure Cloud Service created in step 2.
+
+Once you've satisfied the prerequisites, it's time to configure the Trusted Application source code and deploy it to the Azure Cloud Service created in Step 2.
 
 If you plan on pointing a custom domain or subdomain to your `*.cloudapp.net` server, use that FQDN as your base URL. Otherwise, use `resourcename.cloudapp.net` as your base URL, as created in step 2.
 
 ### Trusted Application Agent
+
 1. Clone code from the samples repo.
 2. Build the solution (NuGet packages will be restored)
```
Comment (lindalu-MSFT) [inline Skype/Trusted-Application-API/samples/AnonMeetingJoinSamples/README.md:52]:
```suggestion
2. Build the solution (NuGet packages will be restored).
```

File: Skype/Trusted-Application-API/samples/AnonMeetingJoinSamples/README.md [hunk 17]
```diff
@@ -4,45 +4,58 @@ description: The Trusted Application API is a Rest API that enables developers t
...
 1. Clone code from the samples repo.
-2. Build the solution (NuGet packages will be restored)
+2. Build the solution (NuGet packages will be restored).
 3. Edit the `ServiceConfiguration.Prod.cscfg` file, substituting `[parameters]` as follows:
+
     ```xml
     <ConfigurationSettings>
       <!-- Replace base.url with your app's url; as registered on Azure -->
-	    <!--This is optional. only needed if you want to leverage the AzureDiagnosticLogger which log all traces in azure diagnostic storage table
+      <!--This is optional. only needed if you want to leverage the AzureDiagnosticLogger which log all traces in azure diagnostic storage table
```
Comment (lindalu-MSFT) [inline Skype/Trusted-Application-API/samples/AnonMeetingJoinSamples/README.md:58]:
```suggestion
      <!--Optional. Only needed if you want to leverage the AzureDiagnosticLogger which log all traces in azure diagnostic storage table.
```

File: Skype/Trusted-Application-API/samples/AnonMeetingJoinSamples/README.md [hunk 18]
```diff
@@ -4,45 +4,58 @@ description: The Trusted Application API is a Rest API that enables developers t
...
-2. Build the solution (NuGet packages will be restored)
+2. Build the solution (NuGet packages will be restored).
 3. Edit the `ServiceConfiguration.Prod.cscfg` file, substituting `[parameters]` as follows:
+
     ```xml
     <ConfigurationSettings>
       <!-- Replace base.url with your app's url; as registered on Azure -->
-	    <!--This is optional. only needed if you want to leverage the AzureDiagnosticLogger which log all traces in azure diagnostic storage table
+      <!--Optional. Only needed if you want to leverage the AzureDiagnosticLogger which log all traces in azure diagnostic storage table.
       if you have other storage to store your trace, this is not needed. -->
```
Comment (lindalu-MSFT) [inline Skype/Trusted-Application-API/samples/AnonMeetingJoinSamples/README.md:59]:
```suggestion
      If you have other storage to store your trace, this is not needed. -->
```

File: Skype/Trusted-Application-API/samples/AnonMeetingJoinSamples/README.md [hunk 19]
```diff
@@ -58,22 +71,27 @@ If you plan on pointing a custom domain or subdomain to your `*.cloudapp.net` se
...
+
+5. Publish the application (right-click `PlatformServiceSamplesAzureService` and select *Publish...*) using the *Production* slot
 
 When visiting your base URL (e.g. `https://name.cloudapp.net`), the expected response is a simple 403.
 
 ### Client webpage
-You will need to modify the `WebsiteSamples\scripts\index.js` file and replace `https://[name].cloudapp.net` with your trusted agent's base URL.
-You will need to modify the `WebsiteSamples\Samples\scripts\sign-in.js' file and replace InviteTargetUri 'toshm@metio.onmicrsoft.com' with your valid user in your own tenant.
+
+You will need to modify the `WebsiteSamples\scripts\index.js` file and replace `https://[name].cloudapp.net` with your trusted agent's base URL. You will need to modify the `WebsiteSamples\Samples\scripts\sign-in.js' file and replace InviteTargetUri 'toshm@metio.onmicrsoft.com' with your valid user in your own tenant.
```
Comment (lindalu-MSFT) [inline Skype/Trusted-Application-API/samples/AnonMeetingJoinSamples/README.md:89]:
```suggestion
You'll need to modify the `WebsiteSamples\scripts\index.js` file and replace `https://[name].cloudapp.net` with your trusted agent's base URL. You'll also need to modify the `WebsiteSamples\Samples\scripts\sign-in.js' file and replace InviteTargetUri 'toshm@metio.onmicrsoft.com' with your valid user in your own tenant.
```

**Pull request merged**